recipes/fastapi-third-party-auth/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set python_min = "3.8" %}
+{% set name = "fastapi-third-party-auth" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 01afc17ca5c6b0a80782805aa5cfdedbae3b26db52d47e1a5d87a50e9e1ae44a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >={{ python_min }}
+    - poetry >=0.12
+    - poetry-core
+    - pip
+  run:
+    - python >={{ python_min }}
+    - fastapi >=0.61.0
+    - pydantic >=1.6.1
+    - cachetools >=4.1.1
+    - requests >=2.24.0
+    - python-jose >=3.2.0
```

Comment:
`python-jose` upstream adds an extra: `python-jose = {extras = ["cryptography"], version = ">= 3.2.0"}`. That extra pulls in `cryptography >=3.4.0`:
https://github.com/mpdavis/python-jose/blob/master/setup.cfg#L45C5-L45C25

So I would add that here as an additional requirement for `run`.